<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaybackVideoBar</class>
 <widget class="QWidget" name="PlaybackVideoBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>60</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="widgetBackground" native="true">
     <property name="styleSheet">
      <string notr="true">#widgetBackground
{
	background-color: rgba(46, 46, 46, 80%);
}
QToolButton
{
    padding-left: 5px;
    padding-right: 5px;
    border: 0px;
    color: rgb(255, 255, 255);
    background-color: transparent;
}
QToolButton:disabled
{
    background-color: transparent;
}
QToolButton:hover
{
    background-color: rgba(44, 122, 158, 220);
}
QToolButton:pressed
{
    background-color: rgba(44, 122, 158, 150);
}
QToolButton:focus
{
    background-color: rgba(44, 122, 158, 220);
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="MyToolButton" name="toolButtonSmartSearch">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resource/resource.qrc">
          <normaloff>:/PlaybackVideoBar/PlaybackVideoBar/smart_search-white.png</normaloff>
          <disabledoff>:/PlaybackVideoBar/PlaybackVideoBar/smart_search-gray.png</disabledoff>:/PlaybackVideoBar/PlaybackVideoBar/smart_search-white.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyToolButton" name="toolButtonFisheye">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resource/resource.qrc">
          <normaloff>:/PlaybackVideoBar/PlaybackVideoBar/fisheye_playback-white.png</normaloff>
          <disabledoff>:/PlaybackVideoBar/PlaybackVideoBar/fisheye_playback-gray.png</disabledoff>:/PlaybackVideoBar/PlaybackVideoBar/fisheye_playback-white.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyToolButton" name="toolButtonZoom">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resource/resource.qrc">
          <normaloff>:/PlaybackVideoBar/PlaybackVideoBar/zoom.png</normaloff>
          <disabledoff>:/PlaybackVideoBar/PlaybackVideoBar/zoom-gray.png</disabledoff>:/PlaybackVideoBar/PlaybackVideoBar/zoom.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyToolButton" name="toolButtonSnapshot">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resource/resource.qrc">
          <normaloff>:/PlaybackVideoBar/PlaybackVideoBar/snapshot.png</normaloff>
          <disabledoff>:/PlaybackVideoBar/PlaybackVideoBar/snapshot-gray.png</disabledoff>:/PlaybackVideoBar/PlaybackVideoBar/snapshot.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyToolButton" name="toolButtonAudio">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resource/resource.qrc">
          <normaloff>:/PlaybackVideoBar/PlaybackVideoBar/audioOff.png</normaloff>
          <disabledoff>:/PlaybackVideoBar/PlaybackVideoBar/audioOff-gray.png</disabledoff>:/PlaybackVideoBar/PlaybackVideoBar/audioOff.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyToolButton" name="toolButtonLock">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resource/resource.qrc">
          <normaloff>:/PlaybackVideoBar/PlaybackVideoBar/lock-white.png</normaloff>
          <disabledoff>:/PlaybackVideoBar/PlaybackVideoBar/lock-gray.png</disabledoff>:/PlaybackVideoBar/PlaybackVideoBar/lock-white.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyToolButton" name="toolButtonQuickTag">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resource/resource.qrc">
          <normaloff>:/PlaybackVideoBar/PlaybackVideoBar/quicktag-white.png</normaloff>
          <disabledoff>:/PlaybackVideoBar/PlaybackVideoBar/quicktag-gray.png</disabledoff>:/PlaybackVideoBar/PlaybackVideoBar/quicktag-white.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyToolButton" name="toolButtonCustomTag">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resource/resource.qrc">
          <normaloff>:/PlaybackVideoBar/PlaybackVideoBar/tag-white.png</normaloff>
          <disabledoff>:/PlaybackVideoBar/PlaybackVideoBar/tag-gray.png</disabledoff>:/PlaybackVideoBar/PlaybackVideoBar/tag-white.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyToolButton" name="toolButtonClose">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resource/resource.qrc">
          <normaloff>:/PlaybackVideoBar/PlaybackVideoBar/close.png</normaloff>
          <disabledoff>:/PlaybackVideoBar/PlaybackVideoBar/close-gray.png</disabledoff>:/PlaybackVideoBar/PlaybackVideoBar/close.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>60</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyToolButton</class>
   <extends>QToolButton</extends>
   <header>mytoolbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
